backend/app/usecase/url/urlretriever.go

```diff
@@ -58,15 +57,8 @@ func (r RetrieverPersist) getURL(alias string) (entity.URL, error) {
 	return url, nil
 }
 
-// GetURLs retrieves URLs created by given user from persistent storage
-func (r RetrieverPersist) GetURLs(user *entity.User) ([]entity.URL, error) {
-	if user == nil {
-		return nil, errors.New("fail to find the owner of the URLs")
-	}
-	return r.getURLsByUser(*user)
```

Comment:
I wonder what happens to this?

Reply:
viewer returns object of type entity.user now (https://github.com/short-d/short/pull/638/files#diff-7f09a888a7007a4d8968cb85f5d8d2beR10) which will be used to fetch user details in AuthQuery Resolver, whether user is nil or not (authToken is valid or not) is handled internally in the auth usecase.

Reply:
@kapillamba4 Cool!